src/app/qgisapp.cpp

```diff
@@ -9192,6 +9192,22 @@ void QgisApp::saveStyleFile( QgsMapLayer *layer )
 
             QgsVectorLayerSaveStyleDialog::SaveToDbSettings dbSettings = dlg.saveToDbSettings();
 
+            QString errorMessage;
+            if ( QgsProviderRegistry::instance()->styleExists( vlayer->providerType(), vlayer->source(), dbSettings.name, errorMessage ) )
+            {
+              if ( QMessageBox::question( nullptr, QObject::tr( "Save style in database" ),
```

Comment:
You could get rid of QObject::tr() in favor of tr() here. I think it'll also result in better classification of the string to translate in transifex et cie.